### ISSUE TITLE
Shadowcljs shows error when running calva.loadFile command

### ISSUE THREAD

**Issue by pinkfrog9** [BetterThanTomorrow/calva#1670]
When running `calva.loadFile`, shadow-cljs will err with:
```
[2022-04-07 23:03:41.704 - WARNING] :shadow.cljs.devtools.server.worker.impl/cljs-compile-ex - {:input {:code "(clojure.core/refer-clojure)", :ns starter.browser, :repl true}}
ExceptionInfo Failed to process REPL command {:eof? false, :ns starter.browser, :form (clojure.core/refer-clojure), :source "(clojure.core/refer-clojure)", :tag :shadow.cljs.repl/process-ex}
        shadow.cljs.repl/process-read-result (repl.clj:502)
        shadow.cljs.repl/process-read-result (repl.clj:476)
        shadow.cljs.repl/process-input (repl.clj:661)
        shadow.cljs.repl/process-input (repl.clj:639)
        shadow.cljs.devtools.server.worker.impl/fn--14599 (impl.clj:698)
        shadow.cljs.devtools.server.worker.impl/fn--14599 (impl.clj:688)
        clojure.lang.MultiFn.invoke (MultiFn.java:234)
        shadow.cljs.devtools.server.util/server-thread/fn--14273/fn--14274/fn--14282 (util.clj:269)
        shadow.cljs.devtools.server.util/server-thread/fn--14273/fn--14274 (util.clj:268)
        shadow.cljs.devtools.server.util/server-thread/fn--14273 (util.clj:241)
        java.lang.Thread.run (Thread.java:833)
Caused by:
ExceptionInfo ns* not supported (require, require-macros, import, import-macros, ... must be part of your ns form) {:rename-macros nil, :renames {}, :use-macros nil, :excludes #{}, :name starter.browser, :op :ns*, :env {:fn-scope [], :locals {}, :js-globals {console {:op :js-var, :name console,
```

**Comment by PEZ** [comment]
Thanks! I'll fix.

FYI. Things still work.

**Comment by riotrah** [comment]
Sorry to reopen if inappropriate, but I get an error like this one all the time, and similarly, I believe nothing is actually broken. Are these similar/related?

```
[2022-04-17 15:17:03.945 - WARNING] :shadow.cljs.devtools.server.worker.impl/cljs-compile-ex - {:input {:code "(load-file \"/\")", :ns cljs.user, :repl true}}
ExceptionInfo Failed to process REPL command {:eof? false, :ns cljs.user, :form (load-file "/"), :source "(load-file \"/\")", :tag :shadow.cljs.repl/process-ex}
```

**Comment by PEZ** [comment]
Can you make a mini-project that exposes this, @riotrah ? Doubt it is related, but you can start with trying some earlier version of Calva (see changelog, these changes are pretty recent) and check if that makes the errors go away. Regardless, I think that a new issue is the way to go, at least if you can reproduce it in some project you can share.

**Comment by bpringe** [comment]
The only place I see Calva running `load-file` code like that is here: https://github.com/BetterThanTomorrow/calva/blob/e42fa3ef148dab9507e9e36e9188849024840e6a/src/testRunner.ts#L269. And that function is only run from `runNamespaceTests`. 

@riotrah Are you by any chance calling that command? I don't think it's useful with cljs, but just wondering. Then again, that might not be coming from Calva anyway.

**Comment by sam-pitch** [comment]
Was able to recreate this warning. The key thing is that it appears just when using an older version of shadow-cljs. I confirmed you get the warning using `thheller/shadow-cljs {:mvn/version "2.15.12"}` but I do not get the warning for the latest (at the time of writting) `thheller/shadow-cljs {:mvn/version "2.19.0"}`

TLDR; Upgrade shadow-cljs to the latest version.
